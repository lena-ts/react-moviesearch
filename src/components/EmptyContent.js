import React from 'react';

const EmptyContent = () => {
  return <div className="no-films">No films found</div>;
};

export default EmptyContent;
