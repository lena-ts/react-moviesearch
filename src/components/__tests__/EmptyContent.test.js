import React from 'react';
import EmptyContent from '../EmptyContent';
import renderer from 'react-test-renderer';

it('Empty content', () => {
  const EmptyContentSnapshot = renderer.create(<EmptyContent />).toJSON();
  expect(EmptyContentSnapshot).toMatchSnapshot();
});
