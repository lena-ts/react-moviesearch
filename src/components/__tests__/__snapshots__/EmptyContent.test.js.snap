// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty content 1`] = `
<div
  className="no-films"
>
  No films found
</div>
`;
